backend/src/facebook-feed/facebook-feed.service.ts

```diff
@@ -0,0 +1,60 @@
+import { Injectable, Logger } from '@nestjs/common';
+import { ConfigService } from '../config/config.service';
+import sources from './facebook-feed-sources';
+import { InjectRepository } from '@nestjs/typeorm';
+import { TransactionRepository, Repository, Transaction } from 'typeorm';
+import { FacebookFeed } from './facebook-feed.entity';
+const config = new ConfigService();
+const axios = require('axios');
+
+@Injectable()
+export class FacebookFeedService {
+
+    constructor(
+        @InjectRepository(FacebookFeed)
+        @TransactionRepository(FacebookFeed)
+        private readonly facebookFeedRepository: Repository<FacebookFeed>,
+        private readonly logger: Logger) {
+        this.logger = new Logger('FacebookFeedService');
+    }
+
+    async onModuleInit() {
+        await this.fetchFacebookFeeds();
+    }
+
+    @Transaction()
+    async fetchFacebookFeeds() {
+        try {
+            const vantaaFacebookDataUrl = `https://graph.facebook.com/v3.1/${sources.vantaa}/posts?`
+                + `fields=id,name,message,story,picture,message_tags,story_tags,caption,`
+                + `created_time,description,place,properties,status_type,type,comments,reactions`
+                + `&access_token=${config.facebookPageToken}`;
+            const facebookFeeds = this.transformData((await axios.get(vantaaFacebookDataUrl)).data.data);
+            await this.facebookFeedRepository.save(facebookFeeds);
+        } catch (error) {
+            this.logger.error(`Failed to get facebook feeds: ${error.message}`)
+        }
+    }
+
+    transformData = feeds => {
```

Comment:
This looks good but I think you can use `lodash` for this type of transformation
We dont have bundle issues, and these code are trivial to write (you already wrote it), so it is maintainable.